ocaml/xenops/device.ml

```diff
@@ -1775,27 +1775,31 @@ let cmdline_of_disp info =
 	in
 	let videoram_opt = ["-videoram"; string_of_int info.video_mib] in
 	let dom0_input_opts x =
-	  (maybe_with_default [] (fun i -> ["-dom0-input"; string_of_int i]) x)
+		(maybe_with_default [] (fun i -> ["-dom0-input"; string_of_int i]) x)
+	in
+	let vnc_opts_of ip_addr_opt auto port keymap =
```

Comment:
Since this is a backport, lets keep the commits and commit history intact as we have in trunk.